docs/api/skills/abstract_round_abci/base.md

```diff
@@ -2093,11 +2105,21 @@ Observe timestamp from last block.
 #### cleanup
 
 ```python
-def cleanup(cleanup_history_depth: int) -> None
+def cleanup(cleanup_history_depth: int, cleanup_history_depth_current: Optional[int] = None) -> None
```

Comment:
if you need more than two underscores for your variable naming, it's probably a bad idea :smiling_face_with_tear: